CWE/decisions/claim_greater.txt

```diff
@@ -5438,15 +5437,8 @@ NOT = { government = democracy government = hms_government }
 			badboy = 12
 			set_country_flag = claim_greater_africa
 			any_country = {
-				limit = {
-					OR = {
-						tag = EGY tag = SUD tag = LBY tag = TUN tag = ALG tag = MOR
-						capital_scope = { continent = africa }
-					}
-				}
-				all_core = { add_core = XAU }
+				any_owned = { limit = { NOT = { is_core = XAU } OR = { continent = africa is_core = EGY is_core = SUD is_core = SSU is_core = LBY is_core = TUN is_core = ALG is_core = MOR is_core = MAU } } add_core = XAU } 
```

Comment:
all_core = { limit = { NOT = { is_core = XAU } OR = { continent = africa is_core = EGY is_core = SUD is_core = SSU is_core = LBY is_core = TUN is_core = ALG is_core = MOR is_core = MAU } } add_core = XAU }